Triangulation_3/doc/Triangulation_3/CGAL/Triangulation_3.h

```diff
@@ -1136,6 +1156,31 @@ Point_iterator points_end() const;
 
 /// @} 
 
+/*! \name Ranges
+
+In order to write \cpp 11 `for`-loops we provide a range type and member functions to generate ranges.
+Note that vertex and cell ranges are special.
+
+*/
+
+/// @{
+/*!
+  returns a range of iterators over all vertices (even the infinite one).
+  Returns an empty range when `t.number_of_vertices() == 0`. 
```

Comment:
There's an inconsistency: for the 3D triangulation, `number_of_vertices()` is actually the number of finite vertices (`tds.vertices().size() - 1`)

I would remove this comment overall (and similar).